.github/workflows/pre_release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Canary Release
+
+on:
+  push:
+    branches:
+    - "main"
+    - "release-v*"
```

Comment:
Shouldn't the canary release only be built from the main branch?

Reply:
Right, update it later. Thanks!